Final Project Submission/Calvey_Final.cpp

```diff
@@ -0,0 +1,214 @@
+// Final Project
+// The task of this program is to use the information I have learned through out the semester and use it for a game.
+// Robert Calvey
+// 30 April 2019
+
+#include <iostream>
+using namespace std;
+
+int  main() {
+string name;
+  int choice;
+  const int start =1;
```

Comment:
good use of constants, but 
- they might as well be globals
- please name them in ALL_CAPS, e.g. `const int TURTLE = 9;`